src/components/FieldNote/FieldNote.module.css

```diff
@@ -12,29 +12,28 @@
   @mixin eds-theme-typography-body-text-xs;
   margin-top: var(--eds-size-half);
   color: var(--eds-theme-color-text-neutral-subtle);
-
-  /**
-     * Field note inside a field that has an error 
-     */
-  &.eds-is-error {
```

Comment:
I always found this naming confusing.. it sounds like a utility class.

Reply:
Yea not sure what the intention was with these